.changeset/silly-flies-care.md

```diff
@@ -0,0 +1,5 @@
+---
+"@pnpm/plugin-commands-script-runners": minor
+---
+
+feat: support wildcard selector to specify multiple scripts to execute"
```

Comment:
add more details. This text is going to be on the release page.

Reply:
[4802e46](https://github.com/pnpm/pnpm/pull/5871/commits/4802e465b66d9e8e1fd828919b7990bdcec4b73b) :+1: